changelog.d/20231120_134843_klakhov_improve_invite_users.md

```diff
@@ -0,0 +1,5 @@
+### Added
+
+- Implemented a feature that allows to invite users to organization using email
```

Comment:
Didn't this feature already exist?

Reply:
Acually you could invite only unregistered users using email and there were no UI for "internal" organization invitations for registered users.

Could you suggest better changelog description for the feature then?

Reply:
Ok, let me explain a bit more.

Before this PR:
1. Organization staff invites user by entering his email. There are two cases: user already exists or doesnt exist. In first case invitation is auto accepted. In second one he receives email letter with registration link.
2. User follows the link and setups his account using registration form. Upon successful registration invitation to such organization is auto accepted.

After this PR (with enabled ORG_INVITATION_CONFIRM in settings):
1. Organization staff invites user by entering his email. There are two cases: user already exists or doesnt exist. In both cases email letter with login link is sent. 
2. User follows the link and logins/registers depending on if he had cvat account or not. Then he is redirected to page with list of his invitations where he can press a button to accept/reject the invitation.
3. Registered users doesn't really need to follow the link in the email, they can view and accept/reject their invitations using our UI. There is new section in Organization top menu called `invitations`. Accordingly, this is a page with a list of all invitations. User can check his invitations manually by visiting this page or by following a link in a popup notification when new invitation is received.

Reply:
> with enabled `ORG_INVITATION_CONFIRM` in settings

Say... how would one actually enable this setting? It seems to be hardcoded to "No".

Reply:
By default it is disabled(like sending emails). I suppose we should enable it in our private production config alongside with email backend setup.

If someone wants to test it in development env, the setting must be changed manually.

Reply:
Okay, how about this:

> When the `ORG_INVITATION_CONFIRM` setting is enabled, organization invitations for existing users are no longer accepted automatically. Instead, the invitee can now review the invitation and accept or decline it.

Reply:
Sounds good to me. With such wording `Changed` category suits more. Updated.